README.md

```diff
@@ -70,23 +70,23 @@ Or [download a ZIP of the files](https://github.com/digininja/DVWA/archive/maste
 - [Installing DVWA on Windows using XAMPP](https://youtu.be/Yzksa_WjnY0)
 - [Installing Damn Vulnerable Web Application (DVWA) on Windows 10](https://www.youtube.com/watch?v=cak2lQvBRAo)
 
-### Windows + XAMPP
-
-The easiest way to install DVWA is to download and install [XAMPP](https://www.apachefriends.org/) if you do not already have a web server setup.
+### Docker
 
-XAMPP is a very easy to install Apache Distribution for Linux, Solaris, Windows and Mac OS X. The package includes the Apache web server, MySQL, PHP, Perl, a FTP server and phpMyAdmin.
+The easiest way to run DVWA is with containers.
```

Comment:
I'm still not going to allow this through. Give people point and click in Windows or a command line, and the majority will find the Windows version easier.